README.md

```diff
@@ -123,6 +123,17 @@ If you want to disable the cronjob completely, run:
 
 To reenable it again simply set the `nextcloud.cron-interval` snap variable to a value that isn't `-1`
 
+#### Nextcloud Prettylink configuration
+
+Before enabling this feature, please first set the correct `overwrite.cli.url` and `htaccess.RewriteBase` in Nextcloud by following [this doc](https://docs.nextcloud.com/server/latest/admin_manual/installation/source_installation.html#pretty-urls)
+
+Afterwards, if desired the snap can remove the index.php from the URL:
+
+    $ sudo snap set nextcloud nextcloud.prettylink=true
```

Comment:
Why don't you call the setting `nextcloud.prettyurls`?

Reply:
thats a good idea! I'll do that 👍

Reply:
done with 5bad0f0

Reply:
Again, why not `nextcloud.prettyurls` instead  of `nextcloud.prettyurl`?

Reply:
sry, I haven't read accurate enough. should be done with 1e2a99a